### PR TITLE
feat(paddle frontend): Added alpha dropout

### PR DESCRIPTION
**Pull request by teckno**
https://github.com/unifyai/ivy/issues/22124

# PR Description

## Related Issue

Close #

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/common.py [hunk 1]
```diff
@@ -4,6 +4,30 @@
...
 
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def alpha_dropout(x, p=0.5, training=True, name=None):
+    if training:
+        alpha = 1.67326
+        scale = 1.0507
+        keep_prob = 1 - p
+        random_values = ivy.random_normal(shape=x.shape, dtype=x.dtype)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/paddle/nn/functional/common.py:14]: With randomization we won't be able to get exact results here you can refer to how other `Selu` activation functions are implemented in frontends.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py [hunk 2]
```diff
@@ -248,6 +248,53 @@ def test_linear(
...
+# alpha_dropout
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.alpha_dropout",
+    d_type_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        shared_dtype=True,
+        min_value=2,
+        max_value=5,
+        min_num_dims=1,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:260]: The function doesn't only work for `1-Dim` Arrays we need to test other dimensions as well

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py [hunk 3]
```diff
@@ -248,6 +248,53 @@ def test_linear(
...
+        max_num_dims=1,
+        min_dim_size=2,
+    ),
+    p=st.floats(min_value=0.0, max_value=1.0),
+    axis=st.integers(min_value=0, max_value=1),
+    training=st.booleans(),
+    mode=st.sampled_from(["upscale_in_train", "downscale_in_infer"]),
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:267]: This mode wasn't indicated in the docs we need to test function based on Documentation
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/alpha_dropout_en.html#alpha-dropout

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below🤞

**Comment by Sarvesh-Kesharwani** [comment]
Closing the PR for the time being, since there has been no activity for more than 2 weeks.
